.codepipeline/buildspec-tests-build.yml

```diff
@@ -47,15 +46,21 @@ phases:
       - . $VENV_TESTS/bin/activate
       - chmod +x environment.sh
       - . ./environment.sh
-      - export ENVIRONMENT=preview
       - python3.9 -m pip install --upgrade pip wheel setuptools
       - make bootstrap
 
   build:
     on-failure: CONTINUE
     commands:
+      # Need to add CBC integration into the list of test reports, in the proper place, if it is executed - hence the below
+      - |
+        if [ "$ENABLE_CBC_INTEGRATION_TESTS" = "true" ]; then
```

Comment:
There's often a danger of relying on strings being the value "true" rather than relying on them being a truthy value.  I this case I would probably just check that the variable is set to something

Reply:
I would agree with you, but the problem is that CodePipeline requires it to be set to _something_. I've used "true" and "false" as a standard (despite the fact that yes, it does really only matter if it's "true", or not "true") - but CodePipeline simply won't let you leave it as an empty string - therefore you'd find that the above would _always_ be truthy. I agree - I wanted to do the same, it's what I see as a slight oversight in the way CodePipeline works, tbh.

Reply:
However, point taken, I'll add that in as a comment above the command because it does seem worth highlighting